src/test/java/app/services/impl/GameServiceImplTest.java

```diff
@@ -0,0 +1,135 @@
+                "Indonesia, Thailand, India, Kenya, Macedonia, Croatia, Macau, Paraguay, Peru, Armenia, " +
+                "Philippines, Vietnam, Jordan and Kuwait.\u003C/h1\u003E\n\u003Ch1\u003E" +
+                "1 IPHONE PAID GAME in more countries than we can count!\u003C/h1\u003E\n\u003Cp\u003E" +
+                "Terms of Use: http://www.rovio.com/eula\u003Cbr /\u003E\n" +
+                "Privacy Policy: http://www.rovio.com/privacy\u003Cbr /\u003E\n" +
+                "This application may require internet connectivity and subsequent data transfer charges may apply." +
+                "\u003Cbr /\u003E\nImportant Message for Parents\u003Cbr /\u003E\n" +
+                "This game may include:\u003Cbr /\u003E\n" +
+                "- Direct links to social networking websites that are intended for an audience over the age of 13." +
+                "\u003Cbr /\u003E\n- Direct links to the internet that can take players away from the game with the potential to browse any web page." +
+                "\u003Cbr /\u003E\n- Advertising of Rovio products and also products from select partners." +
+                "\u003Cbr /\u003E\n- The option to make in-app purchases. The bill payer should always be consulted beforehand.\u003C/p\u003E";
+
+        gameDTO = new GameDTO(
+                144L,
+                "Angry Birds",
+                LocalDate.of(2009, 12, 11),
+                "https://media.rawg.io/media/games/fc8/fc839beb76bd63c2a5b176c46bdb7681.jpg",
+                80.0,
+                4,
+                description,
+                parentPlatFormDTOSet,
+                genreDTOSet
+        );
+    }
+
+    @Test
+    void getAllGames() {
+        int expectedSize = 25;
+
+        Set<GameDTO> allGames = gameService.getAllGames();
+
+        assertNotNull(allGames);
+        assertThat(allGames.size(), is(expectedSize));
+    }
+
+    @Test
+    void getAllGamesWithDetails() {
+        int expectedSize = 25;
+
+        Set<GameDTO> allGames = gameService.getAllGames();
+        Set<GameDTO> allGamesWithDetails = gameService.getAllGamesWithDetails(allGames);
+        Optional<GameDTO> gameDTOWithDetails = allGamesWithDetails.stream().findFirst();
+
+        assertThat(allGamesWithDetails.size(), is(expectedSize));
+        assertThat(gameDTOWithDetails.isPresent(), is(true));
+        gameDTOWithDetails.ifPresent(game ->
+                assertThat(game.getDescription(), notNullValue())
+        );
+    }
+
+    @Test
+    void getGameById() {
+        Long expectedId = gameDTO.getId();
+        GameDTO expectedGameDTO = gameDTO;
+
+        GameDTO actualGameDTO = gameService.getGameById(expectedId);
+
+        assertNotNull(actualGameDTO);
+        assertThat(actualGameDTO.getId(), is(expectedId));
+        assertThat(actualGameDTO.getDescription(), equalTo(expectedGameDTO.getDescription()));
+        assertThat(actualGameDTO.toString(), equalTo(expectedGameDTO.toString()));
```

Comment:
This should probably be changed to `assertThat(actualGameDTO, equalTo(expectedGameDTO))` to do a more thorough comparison

Reply:
The toString wasn't supposed to be there anyway - will remove it